packages/toolpad-core/src/Account/AccountPreview.tsx

```diff
@@ -99,13 +99,19 @@ function AccountPreview(props: AccountPreviewProps) {
   if (variant === 'expanded') {
     return (
       <Stack direction="row" justifyContent="space-between" sx={{ py: 1, px: 2, gap: 2, ...sx }}>
-        <Stack direction="row" justifyContent="flex-start" spacing={2}>
+        <Stack direction="row" justifyContent="flex-start" spacing={2} overflow="hidden">
           {avatarContent}
           <Stack direction="column" justifyContent="space-evenly">
-            <Typography variant="body2" fontWeight="bolder" noWrap>
+            <Typography
+              variant="body2"
+              fontWeight="bolder"
+              // Only truncate when a longer text causes the text to overflow over the MoreIconButton
+              maxWidth={handleClick ? 180 : 'unset'}
```

Comment:
Looks good, but is there no way to make it overflow with the available width without us having to set a manual value?
I feel like it should be possible but haven't tried it.

Also what is the behavior if there is no options button? Does it truncate with an ellipsis after the sidebar width? I guess that's what it should do in every case...

Reply:
The idea to only truncate when there is an options button is to allow the popover to have the complete information, so something like:

https://github.com/user-attachments/assets/9848b2ee-becf-401e-b3a6-fb35a32a923d

We could truncate in all cases, but then do you suggest including a Tooltip when the text is inside the popover; something like: 

https://github.com/user-attachments/assets/37583309-75c9-45b2-9529-28a8f9715f36

Reply:
I meant that if you don't pass a `handleClick` and so there's no options button to show a popover, this is what it looks right now:

<img width="837" alt="Screenshot 2025-01-07 at 10 44 02" src="https://github.com/user-attachments/assets/82c4ab19-fdaf-4e8f-a216-4f834fdd134e" />

So maybe all these overflows should ellipsize automatically? Probably not a huge deal though, it was just something I noticed.

Reply:
Ah got it! Yes the ellipsis should happen in this case as well, good catch